Dockerfile

```diff
@@ -1,8 +1,8 @@
 # syntax=docker/dockerfile:1
 
-ARG CLOUDFLARED_VERSION=2023.3.1
+ARG CLOUDFLARED_VERSION=2023.4.0
 ARG ALPINE_VERSION=3.17
-ARG GO_VERSION=1.19
+ARG GO_VERSION=1.19.6
```

Comment:
No need to set the patch release. `1.19` takes the very latest Go 1.19.

Reply:
removed.